### DISCUSSION TITLE
I would like to ask if you could provide official support for 'reCaptcha' on the login form?

### DISCUSSION THREAD

**Discussion by ariferol** [keycloak/keycloak#32044] [Ideas]
I see that the registration form has official support for 'reCaptcha.' However, I didn't see this support on the login form. I have seen some external solutions (Sample: https://github.com/raptor-group/keycloak-login-recaptcha). Is it possible to officially add optional 'reCaptcha' support to the login form?

**Comment by ahus1** [comment]
I see that there is a long-outstanding issue for that: https://github.com/keycloak/keycloak/issues/14936

Unfortunately it is tagged as a task, where it should be an enhancement. Please up-vote that issue, and possibly advertise for this issue so it gets more votes.
